src/doc/src/guide/project-layout.md

```diff
@@ -11,24 +11,24 @@ Cargo [package][def-package]:
 │   ├── lib.rs
 │   ├── main.rs
 │   └── bin/
-│       ├── named-executable.rs
-│       ├── another-executable.rs
-│       └── multi-file-executable/
+│       ├── named_executable.rs
+│       ├── another_executable.rs
+│       └── multi_file_executable/
```

Comment:
I believe this would have end-user affects, making the default binary names use `_` when the status quo is for binaries to use `-`.

As these are not just `mod` names but build target names, i would be inclined to leave all of these as `-`